lib/components/molecules/Form/FieldToggle/FieldToggle.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+import { Field } from 'formik';
+import styles from './FieldToggle.style';
+import type { Props } from './types';
+
+import Label from '../../../atoms/Label/Label';
+import Input from '../../../atoms/Input/Input';
+
+const FieldToggle = ({
+  id,
+  name,
+  disabled,
+  label,
+  indicatorOn,
+  indicatorOff,
+  className,
+  inheritedStyles,
+  displayIndicators,
+  intialToggleState,
+  toggleCheckboxClick,
+  ...otherProps
+}: Props): Node => {
+  const [toggleOn, handleToggle] = React.useState(intialToggleState);
```

Comment:
Please change this toggleOn to toggle as the variable can have both states ON/OFF